client/components/registration/registration.js

```diff
@@ -54,17 +95,25 @@ export default class Registration extends React.Component {
   handleSubmit(event) {
     const {setLoading} = this.context;
     event.preventDefault();
-    const {orgSlug, authenticate, settings, setUserData, userData} = this.props;
+    const {orgSlug, authenticate, settings} = this.props;
```

Comment:
I think this thing breaks the signup process for the organization without any verification. I have tested it for default organization.

Reply:
ok thank you I think I did not test that, I will double check and add a failing test